.github/workflows/CI.yaml

```diff
@@ -35,10 +35,16 @@ jobs:
       - name: Clean neo4j from the cache
         if: success() || failure() # run this step even if previous step failed
         run: (find ~/.gradle/caches -name "*neo4j*" -exec rm -rf {} \;) || echo "All neo4j files cleaned"
+      - name: Check procedures included in apoc full
+        run: git diff --exit-code full/src/main/resources/extended.txt || (echo "extended.txt file does not contain all changes" && exit -1)
       - name: Archive test results
         uses: actions/upload-artifact@v2
+        if: ${{ always() }}
```

Comment:
I googled to understand the meaning of this, and Stack overflow tells me it can probably be simplified to just 
`if: always()`

Reply:
I had to add this line to archive the test results also in case of test failures